### DISCUSSION TITLE
Add option to pause on focused window OR maximised window

### DISCUSSION THREAD

**Discussion by IcyAlmond** [catsout/wallpaper-engine-kde-plugin#175] [Ideas]
Add option to pause if there is EITHER a focused window OR a maximised window.

Currently in multi monitor setups:

* If pause on "focused window" is selected, if a window is maximised, but not focused, the wallpaper will still play
* If pause on "maximised window" is selected, it will not pause if there is just a focused window on that screen.
* If pause on "any window" is selected, it will pause if there is any window on that screen, even if it's not focused

An option to pause the wallpaper if a window is focused on that screen OR if there is a maximised window on that screen would be nice

**Comment by catsout** [comment]
option 'Focus or Maximized' is added.
